test/testdata

```diff
@@ -308,7 +308,7 @@ SPun      Psi4        GenericSPunTest       basicPsi4-1.3.1       dvb_sp_uhf.out
 SPun      Psi4        GenericSPunTest       basicPsi4-1.3.1       dvb_sp_uks.out
 SPun      QChem       GenericSPunTest       basicQChem4.2         dvb_sp_un.out
 SPun      QChem       GenericSPunTest       basicQChem5.1         dvb_sp_un.out
-SPun      Turbomole   GenericSPunTest       basicTurbomole5.9     dvb_un_sp/basis    dvb_un_sp/control    dvb_un_sp/job.last    dvb_un_sp/coord    dvb_un_sp/alpha    dvb_un_sp/beta    dvb_un_sp/gradient
+SPun      Turbomole   GenericSPunTest       basicTurbomole5.9     dvb_un_sp/basis    dvb_un_sp/control    dvb_un_sp/dscf.out    dvb_un_sp/coord    dvb_un_sp/alpha    dvb_un_sp/beta    dvb_un_sp/gradient
```

Comment:
Was this just a renaming of the corresponding file or did you have to rerun the job?

Reply:
It's the same file but edited to remove the optimisation output, leaving just the output from dscf, and then renamed to a more likely name for a single point output file if that makes sense